<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addNewConnectionBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>85</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Добавить
соединение</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delConnectionBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>85</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Удалить
соденинение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delAllConnectionsBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Удалить все
соединения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Отпрвить сообщение</string>
    </property>
   </widget>
   <widget class="QLabel" name="yourIPLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputArea">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chatArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="connectionIPLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP собеседника:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="connectionList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>391</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>171</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>171</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Добавленные соединения:</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
